src/platform/utilities/ui/index.js

```diff
@@ -96,7 +96,7 @@ export function customScrollAndFocus(scrollAndFocusTarget, pageIndex) {
   } else if (typeof scrollAndFocusTarget === 'function') {
     scrollAndFocusTarget(pageIndex);
   } else {
-    scrollToTop('topScrollElement', getScrollOptions());
+    scrollTo('topContentElement', getScrollOptions());
     // h3 should be a unique header on the page
     focusByOrder(['#main h3', defaultFocusSelector]);
```

Comment:
When `useCustomScrollAndFocus: true` is set in the app `config/form.js`, we assume the app has been set up to have unique `H3`s on every page, so we keep that `H3` at the top of the page instead of the "Step x of y" `H2`.